client/components/QuestionListEntry.jsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import Moment from 'react-moment';
 
 class QuestionListEntry extends React.Component {
```

Comment:
Hmm...is there any reason this has to be a stateful component? I think it's best to make a it a function unless you need to store state on it. If you do that elsewhere, I'd change it there too.

Reply:
I appreciate the feedback! I was unsure of whether to do stateful or stateless as some of the functionality I'll need later on will probably use state. I think when I decide where I want to hold that state, I can put your advice to good use and convert the other components to stateless. Thank you!